Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM python:3.10
-WORKDIR /code
-COPY ./requirements.txt /code/requirements.txt
-RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
-COPY ./app /code/app
-COPY ./data /code/data
-CMD ["uvicorn", "app.main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "8013"]
\ No newline at end of file
+WORKDIR /app
+COPY ./requirements.txt requirements.txt
+RUN pip install --no-cache-dir --upgrade -r requirements.txt
+COPY ./app /app
+COPY ./data /data
+ENV PORT 8013
+EXPOSE 8013
+# CMD ["uvicorn", "main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "8013"]
```

Comment:
Is this commented out line intentional?

Reply:
Yes, it's intentional. I commented it exactly as reference in case we need to update the pipeline somewhere. 
Readme is updated now.